pandas/tests/test_groupby.py

```diff
@@ -2481,6 +2481,26 @@ def test_size(self):
             self.assertEqual(result[key], len(group))
 
     def test_count(self):
+        from string import ascii_lowercase
+        n = 1 << 15
+        dr = date_range('2015-08-30', periods=n // 10, freq='T')
+
+        df = DataFrame({
+                '1st':np.random.choice(list(ascii_lowercase), n),
+                '2nd':np.random.randint(0, 5, n),
+                '3rd':np.random.randn(n).round(3),
+                '4th':np.random.randint(-10, 10, n),
+                '5th':np.random.choice(dr, n),
+                '6th':np.random.randn(n).round(3),
+                '7th':np.random.randn(n).round(3)})
+
+        for col in '3rd', '5th', '6th', '7th':
+            df.loc[np.random.choice(n, n // 10), col] = np.nan
+
+        for key in '1st', '2nd', ['1st', '2nd']:
+            left = df.groupby(key).count()
+            right = df.groupby(key).apply(DataFrame.count).drop(key, axis=1)
+            assert_frame_equal(left, right)
```

Comment:
plz read the test; 10% within each column are set to nan at **random**; i do not have the actual numbers.